src/styles/themes.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { DefaultTheme } from '@react-navigation/native';
+import { Platform } from 'react-native';
+
+import { _PRIMARY_COLOR as PRIMARY_COLOR } from '../config';
+
+/**
+ * Light theme color scheme
+ * @type {{
+ * midContrastDetail: string,
+ * errorBgColor: string,
+ * backgroundColor: string,
+ * borderColor: string,
+ * lowContrastDetail: string,
+ * textColorShadow: string,
+ * positiveBalanceColor: string,
+ * textColor: string,
+ * errorTextColor: string,
+ * tabBarBackground: string,
+ * primaryOpacity30: string,
+ * textColorShadowLight: string,
+ * primaryOpacity10: string,
+ * errorTextShadow: string,
+ * primary: string}}
+ * @property {string} backgroundColor The main background color
+ * @property {string} lowContrastDetail Low contrast with background: separator lines, containers...
+ * @property {string} midContrastDetail Medium contrast with background: placeholders, ...
+ * @property {string} borderColor Defines borders
+ * @property {string} textColor Maximum contrast with the background color, for reading
+ * @property {string} textColorShadow Washed down version of the text
+ * @property {string} textColorShadowLight More washed down version of the text
+ * @property {string} tabBarBackground Specific for tab bar selectors
+ * @property {string} positiveBalanceColor Represents a positive feedback for the user
+ * @property {string} errorBgColor For containers with error feedbacks
+ * @property {string} errorTextColor For texts with error messages
+ * @property {string} errorTextShadow Washed down version of error texts
+ * @property {string} primary Primary color, as set on the application config file
+ * @property {string} primaryOpacity10 Primary color washed down to 10% opacity
+ * @property {string} primaryOpacity30 Primary color washed down to 30% opacity
+ */
+export const COLORS = {
+  backgroundColor: '#fff',
+  lowContrastDetail: '#f7f7f7',
+  midContrastDetail: '#9e9e9e',
+  borderColor: '#eee',
+  textColor: '#000',
+  textColorShadow: 'rgba(0, 0, 0, 0.5)',
+  textColorShadowLight: 'rgba(0, 0, 0, 0.3)',
+  tabBarBackground: '#333',
+  positiveBalanceColor: '#0DA0A0',
+  errorBgColor: '#DE3535',
```

Comment:
We used to have these colors in the constants file. Can we delete from there?

Reply:
They are prepared to be deleted on the colors refactor on #327